packages/firebase_remote_config/firebase_remote_config/android/src/main/java/io/flutter/plugins/firebase/firebaseremoteconfig/FirebaseRemoteConfigPlugin.java

```diff
@@ -100,14 +101,14 @@ private FirebaseRemoteConfig getRemoteConfig(Map<String, Object> arguments) {
   }
 
   @Override
-  public void onMethodCall(MethodCall call, final MethodChannel.Result result) {
+  public void onMethodCall(MethodCall call, @NonNull final MethodChannel.Result result) {
     Task<?> methodCallTask;
     FirebaseRemoteConfig remoteConfig = getRemoteConfig(call.arguments());
 
     switch (call.method) {
       case "RemoteConfig#ensureInitialized":
         {
-          methodCallTask = remoteConfig.ensureInitialized();
+          methodCallTask = Tasks.whenAll(remoteConfig.ensureInitialized());
```

Comment:
Currently on Android the task returned by `RemoteConfig.ensureInitialized()` returns a FirebaseRemoteConfigInfo instance which gets sent to the Flutter standard message codec and causes a crash as this isn't a supported value that can be serialized by the codec.

As we don't at this point care about the value of the task result, I've wrapped the task in a [`whenAll`](https://developers.google.com/android/reference/com/google/android/gms/tasks/Tasks#whenAll(com.google.android.gms.tasks.Task%3C?%3E...)) which will just return `void` when completed.